backend/dal/Helpers/Extensions/AcquisitionExtensions.cs

```diff
@@ -40,6 +40,27 @@ private static IQueryable<Entities.PimsAcquisitionFile> GenerateCommonAcquisitio
         {
             filter.ThrowIfNull(nameof(filter));
 
+            if (!string.IsNullOrWhiteSpace(filter.Pid))
+            {
+                var pidValue = filter.Pid.Replace("-", string.Empty).Trim().TrimStart('0');
```

Comment:
This would be in conflict with the pid search that we use on the mapview. Do the requirements mention anything about removing any leading zeroes?

Reply:
This is for search on the DB where the values for both pin and pid are integers. Note that we do the same thing when searching leases by pid/pin

```
CREATE TABLE [dbo].[PIMS_PROPERTY](
...
[PID] [int] NULL,
[PIN] [int] NULL,
...
```